content/sensu-go/5.2/sensuctl/reference.md

```diff
@@ -18,6 +18,7 @@ menu:
 - [Managing resources](#managing-resources)
 - [Time formats](#time-formats)
 - [Shell auto-completion](#shell-auto-completion)
+- [Config files](#)
```

Comment:
Should this have more after the `#`? Maybe `configuration-files`?
```suggestion
- [Config files](#configuration-files)
```

Reply:
Right, I hadn't determined what the link would be when I was writing. Will update with this.